scripts/irods/test/test_iput.py

```diff
@@ -886,6 +887,50 @@ def test_large_put(self):
         self.object_name = 'test_large_put'
         self.run_tests(self.object_name, 5120000)
 
+    def test_iput_with_updated_checksum_read_buffer_size_in_bytes__issue_7947(self):
```

Comment:
Please rework the test's name to describe what it does. For example:
```py
def test_server_honors_server_config_option_checksum_read_buffer_size_in_bytes__issue_7947(self):
```

Reply:
done